pex/testing.py

```diff
@@ -334,6 +336,22 @@ def bootstrap_python_installer(dest):
 PY36 = '3.6.6'
 
 _VERSIONS = (PY27, PY35, PY36)
+# This is the filename of a sentinel file that sits in the pyenv root directory.
+# Its purpose is to indicate whether pyenv has the correct interpreters installed
+# and will be useful for indicating whether we should trigger a reclone to update
+# pyenv.
+INTERPRETER_SET_FINGERPRINT = '_'.join(_VERSIONS) + '_pex_fingerprint'
```

Comment:
How about `_INTERPRETER_SET_FINGERPRINT` to respect the local precedent of private constants?